sql/expression/function/queryinfo.go

```diff
@@ -175,3 +202,34 @@ func (r FoundRows) Children() []sql.Expression {
 func (r FoundRows) WithChildren(children ...sql.Expression) (sql.Expression, error) {
 	return sql.NillaryWithChildren(r, children...)
 }
+
+func toInt64(x interface{}) int64 {
```

Comment:
> Hey @xujihui1985, thanks for sending us this contribution! 🙏
> 
> The code looks good; I just had one small suggestion for reusing some existing int conversion code.
> 
> I think the last thing we need it getting some test coverage. There are already [some tests for `last_update_id()` in script_queries.go](https://github.com/dolthub/go-mysql-server/blob/main/enginetest/queries/script_queries.go#L664) that should be pretty easy to extend.

thanks for reviewing the PR, didn't notice sql.Int64.Convert, toInt64 is copy from sql/plan/insert.go and I just put it here because I have no right place to put it,  I have replace it with sql.Int64.Convert now.
also test code is added, and I have tested in my own dolt branch